src/cluecode/finder.py

```diff
@@ -45,6 +45,21 @@
 """
 
 
+def prefilter_urls_and_emails(key, line):
```

Comment:
could you find a better name and doc? what does this function do? it does not prefilter any thing, right?
It tells whether a line is a likely candidate instead. Also it oucl be best to split this in two functions: one for email and one for urls

Reply:
Fixed, thanks ! :)